cheroot/ssl/builtin.py

```diff
@@ -45,12 +45,43 @@ class BuiltinSSLAdapter(Adapter):
     ciphers = None
     """The ciphers list of SSL."""
 
-    def __init__(self, certificate, private_key, certificate_chain=None, ciphers=None):
+    VERIFY_MODES = {
+        None: ssl.CERT_NONE,
+        'none': ssl.CERT_NONE,
+        'optional': ssl.CERT_OPTIONAL,
+        'required': ssl.CERT_REQUIRED,
+    }
+
+    _verify_mode = None
+
+    @property
+    def verify_mode(self):
+        """Whether peer certificates are validated or not.
+
+        - None, "none": peer cert ignored (ssl.CERT_NONE)
+        - "optional": peer cert optional, but validated if present (ssl.CERT_OPTIONAL)
+        - "required": peer cert required (ssl.CERT_REQUIRED)
+        """
+        return self._verify_mode
+
+    @verify_mode.setter
+    def verify_mode(self, value):
+        if value not in self.VERIFY_MODES:
+            raise ValueError('Invaid verify_mode "%s"; must be one of %s' % (
+                             value, self.VERIFY_MODES.keys()))
+        self._verify_mode = value
```

Comment:
please transform `'none'` string to `None` here, so that `if self.verify_mode:` check would evaluate stable result